sirbot_pyslackers/endpoints/slack/messages.py

```diff
@@ -13,8 +13,15 @@
 from .utils import ADMIN_CHANNEL, HELP_FIELD_DESCRIPTIONS
 
 LOG = logging.getLogger(__name__)
-STOCK_REGEX = re.compile(r"\b(?P<asset_class>[cs])\$(?P<symbol>\^?[A-Z.]{1,5})\b")
+STOCK_REGEX = re.compile(
+    r"\b(?P<asset_class>[cs])\$(?P<symbol>\^?[A-Z.]{1,5})-?(?P<currency>[A-Z]{3})?\b"
```

Comment:
Is your intent to allow both `BTCUSD` and `BTC-USD` formats? I think we should stick with just one and probably use the more standard delimiter for FOREX (`/`). In which case, being a little pedantic, I think a better name for `currency` is `quote` or `quote_currency`.

Reply:
No, that was a side-effect of a poor regex I think :) I intended BTC-USD format, since that's what's returned today, then of course had to make the "-" optional since people will still use just "BTC" without specifying a currency... 

The "-" is used basically as far as I can tell because that's the format Yahoo takes the request in but we can always translate this in the code. 

Would you prefer c$BTC-EUR or c$BTC/EUR ?

Reply:
Set to c$<ticker>-<curr> as this matches the Yahoo format, as well as what is returned in the display today (BTC-USD)